### PR TITLE
Modif utils net

### PR DESCRIPTION
**Pull request by davidmeunier79**

### CODE REVIEW

File: graphpype/utils_net.py [hunk 1]
```diff
@@ -3,129 +3,107 @@
...
-
+        
     return node_corres
 
-
 def read_Pajek_corres_nodes_and_sparse_matrix(Pajek_net_file):
-
+    """
+    reading corresponding vector and sparse matrix from Pajek file
+    """
```
Comment (jasmainak) [inline graphpype/utils_net.py:67]: @davidmeunier79 the docstrings don't seem to follow numpy docstring conventions. Could you maybe try to fix this?

Comment (davidmeunier79) [reply graphpype/utils_net.py:67]: This functions are called by the nodes functions either in "graphpype/nodes" or in "graphpype/interface", this are the one that needs to be documented, and so far this one follows the documentation given by nipype. Normally these functions in "utils_" are not directly called for the definition of pipelines

Comment (jasmainak) [reply graphpype/utils_net.py:67]: If it's not meant to be used by the user, you should add an underscore in the beginning to say that it's a private function.

Comment (davidmeunier79) [reply graphpype/utils_net.py:67]: This is the case of all functions in ephypype as well (the functions at the ephypype level, not in a subdirectory, like aux_tools.py, fif2ts.py, spectral.py, etc.). That could be done, but would involve a great amount of remounding the whole package. Maybe we can plan that for a future release? @annapasca what do you think?

**Pull request merged**